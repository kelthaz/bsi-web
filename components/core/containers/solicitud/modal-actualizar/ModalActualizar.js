import { useRouter } from 'next/router';
import { useDispatch, useSelector } from 'react-redux';
import { useEffect, useState } from 'react';
import Modal from '../../../../shared/modal/Modal';
import { resetChangePage, onHandleSubmit } from '../../../../../redux/actions/formulario';

const ModalActualizar = () => {
  const { push } = useRouter();
  const dispatch = useDispatch();
  const { routePage, showModal, isValid } = useSelector((state) => state.formulario);
  const [openModal, setOpenModal] = useState(false);

  const handleModalFirstOption = () => {
    setOpenModal(false);
    if (isValid) {
      dispatch(onHandleSubmit(true));
    } else {
      dispatch(resetChangePage());
    }
  };

  const handleModalSecondOption = async () => {
    setOpenModal(false);
    dispatch(resetChangePage());
    await push(routePage);
  };

  useEffect(() => {
    if (showModal) {
      setOpenModal(true);
    }
  }, [showModal]);

  return (
    <Modal openModal={openModal} setOpenModal={setOpenModal} onClose={() => dispatch(resetChangePage())}>
      <div className="modal-alerta-container">
        <h4 className="color-blue-storm">
          {isValid
            ? 'Se han detectado cambios en formulario...'
            : 'Se han detectado cambios con errores en el formulario...'}
        </h4>
        <p className="dark-gray body2">
          {isValid
            ? 'Desea guardar los cambios o descartarlos.'
            : 'Desea permanecer y corregir los errores o desea descartar los cambios.'}
        </p>
        <div className="flex-row-center-config">
          <button type="button" className="btn-medium-secondary mr-2" onClick={handleModalFirstOption}>
            {isValid ? 'Sí, guardar' : 'Sí, corregir'}
          </button>

          <button type="button" className="btn-medium ml-2" onClick={handleModalSecondOption}>
            No, descartar
          </button>
        </div>
      </div>
    </Modal>
  );
};

export default ModalActualizar;
