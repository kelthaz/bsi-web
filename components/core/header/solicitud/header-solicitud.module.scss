@import '../../../../styles/scss/modules/var';
@import '../../../../styles/scss/modules/mixins';

.container-header {
  align-items: center;
  border-bottom: 1px solid $blue-shadow;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding: 16px 32px;

  @include media('sm') {
    padding: 14px 16px;
  }
}
