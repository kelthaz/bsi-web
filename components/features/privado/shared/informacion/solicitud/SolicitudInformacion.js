import React from 'react';
import PropTypes from 'prop-types';
import { MORAL } from '../../../../../../constants/persona';

const SolicitudInformacion = ({ informacionSolicitud, tipoPersona }) => {
  const labels = {
    usoCredito: 'Destino del crédito',
    descripcionCredito: 'Razón social',
    domicilioFiscal: 'Domicilio fiscal',
    domicilioComercial: 'Domicilio comercial',
    nombreCompletoRecibe: 'Nombre de quien podría recibir token',
    celularRecibe: 'Teléfono de quien podría recibir token',
    telefonoEmpresa: 'Teléfono del negocio',
    curp: tipoPersona === MORAL ? 'CURP' : 'CURP del representante legal',
    estadoCivil: 'Estado civil del solicitante',
    nombreConyuge: 'Nombre de la Pareja',
    numeroEmpleados: 'Número de empleados',

    montoSolicitado: 'Crédito solicitado',
    montoAprobado: 'Crédito aprobado',

    tasaOrdinaria: 'Tasa ordinaria',
    tasaMoratoria: 'Tasa moratoria',
    comisionApertura: 'Comisión por apertura',
    plazo: 'Plazo del crédito',
    periodicidad: 'Esquema de pago',
    pago: 'Pagos',
    cat: 'CAT',
    tieneCuentaBancoppel: 'Tiene cuenta BanCoppel',
    clabe: 'Número de cuenta BanCoppel',
  };

  if (tipoPersona === MORAL) {
    delete labels.estadoCivil;
    delete labels.nombreConyuge;
  }

  const itemsPerfil = Object.keys(labels);

  return (
    <>
      {itemsPerfil.map((item) => (
        <div key={item} className="row pb-3">
          <div className="col-6">
            <div className="body2">{labels[item]}</div>
          </div>
          <div className="col-6">
            <div className="color-blue-storm body2">{informacionSolicitud[item]}</div>
          </div>
        </div>
      ))}
    </>
  );
};

SolicitudInformacion.propTypes = {
  informacionSolicitud: PropTypes.object.isRequired,
  tipoPersona: PropTypes.string.isRequired,
};

export default SolicitudInformacion;
