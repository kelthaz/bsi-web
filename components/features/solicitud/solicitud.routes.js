import cargaDocumentosRoutes from './carga-documentos/carga-documentos.routes';
import datosEmpresaRoutes from './datos-empresa/datos-empresa.routes';
import datosPersonalesRoutes from './datos-personales/datos-personales.routes';
import documentacionRoutes from './documentacion/documentacion.routes';
import ofertaRoutes from './oferta/oferta.routes';

const solicitudRoutes = [
  ...datosPersonalesRoutes,
  // ...datosEmpresaRoutes,
  // ...documentacionRoutes,
  // ...cargaDocumentosRoutes,
  // ...ofertaRoutes
].map((route) => ({ ...route, feature: 'solicitud' }));

export default solicitudRoutes;
