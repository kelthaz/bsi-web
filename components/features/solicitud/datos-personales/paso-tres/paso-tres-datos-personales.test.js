import '@testing-library/jest-dom';
import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { useRouter } from 'next/router';
import { act } from 'react-dom/test-utils';
import storeTest from '../../../../../redux/storeTest';
import PasoTresDatosPersonales from './PasoTresDatosPersonales';
import { PASO_CUATRO_DATOS_PERSONA_ROUTE } from '../../../../../constants/routes/solicitud/persona';
import SectoresRepositorio from '../../../../../services/simulador/sectores.repositorio';
import { FISICA, MORAL } from '../../../../../constants/persona';
import ModalActualizar from '../../../../core/containers/solicitud/modal-actualizar/ModalActualizar';

jest.mock('../../../../../services/simulador/sectores.repositorio');

describe('Pruebas en el componente PasoTresDatosPersonales', () => {
  const sectores = [
    {
      id: 1,
      nombre: 'SERVICIOS PROFESIONALES Y TÉCNICO',
    },
    {
      id: 2,
      nombre: 'COMERCIO',
    },
    {
      id: 3,
      nombre: 'SERVICIOS FINANCIEROS',
    },
    {
      id: 4,
      nombre: 'CONSTRUCIÓN',
    },
    {
      id: 5,
      nombre: 'RESTO INDUSTRIA',
    },
    {
      id: 6,
      nombre: 'COMUNICACIONES Y TELECOMUNICACIONES',
    },
    {
      id: 7,
      nombre: 'TRANSPORTE',
    },
    {
      id: 8,
      nombre: 'HOTELES Y RESTAURANTES',
    },
    {
      id: 9,
      nombre: 'INDUSTRIA AUTOMOTRIZ',
    },
    {
      id: 10,
      nombre: 'INDUSTRIA QUIMICA Y FARMACÉUTICA',
    },
    {
      id: 11,
      nombre: 'INDUSTRIA MATERIALES DE CONSTRUCCIÓN',
    },
    {
      id: 12,
      nombre: 'INDUSTRIA TEXTIL Y DE CALZADO',
    },
    {
      id: 13,
      nombre: 'SALUD',
    },
  ];

  const giros = [
    {
      id: 1,
      nombre: 'ARRENDAMIENTO DE INMUEBLES COMERCIALES E INDUSTRIALES.',
    },
    {
      id: 2,
      nombre: 'ESTACIONAMIENTOS Y PENSIONES PARA VEHÍCULOS',
    },
    {
      id: 3,
      nombre: 'AGENCIAS PROFESIONALES.',
    },
    {
      id: 4,
      nombre: 'DESPACHOS DE PROFESIONISTAS.',
    },
    {
      id: 5,
      nombre: 'ORGANIZACIONES Y CÁMARAS EMPRESARIALES',
    },
  ];

  const push = jest.fn();
  const beforePopState = jest.fn();
  useRouter.mockImplementation(() => ({ push, beforePopState }));

  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('Debe de mostrarse correctamente el componente', async () => {
    const store = storeTest();
    const wrapper = mount(
      <Provider store={store}>
        <PasoTresDatosPersonales sectores={sectores} validate />
        <ModalActualizar />
      </Provider>
    );
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });

  test('Debe de guardar en el reducer el nuevo paso, nombreEmpresa, sector, giro, descripcionEmpresa y enviarlo al siguiente paso, como persona Fisica', async () => {
    // arrange
    const store = storeTest();
    const {
      solicitud: { currentStep },
    } = store.getState();
    const wrapper = mount(
      <Provider store={store}>
        <PasoTresDatosPersonales sectores={sectores} validate />
        <ModalActualizar />
      </Provider>
    );

    await SectoresRepositorio.getGiroPorSector.mockResolvedValue({ data: giros });

    const form = wrapper.find('form');
    const nombreEmpresa = wrapper.find('input').find({ name: 'nombreEmpresa' });
    const sector = wrapper.find({ name: 'sector' }).find('.item');
    const descripcionEmpresa = wrapper.find('textarea').find({ name: 'descripcionEmpresa' });
    // act
    await act(async () => {
      nombreEmpresa.simulate('change', {
        target: { value: 'Paletas don chucho', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      descripcionEmpresa.simulate('change', {
        target: { value: 'Vende paletas', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      sector.at(1).simulate('click');
    });

    wrapper.update();
    const giro = wrapper.find({ name: 'giro' }).find('.item');

    await act(async () => {
      giro.at(1).simulate('click');
    });

    await act(async () => {
      form.simulate('submit');
    });

    const {
      solicitud: { datosPersonales },
    } = store.getState();
    // // assert
    expect(datosPersonales.nombreEmpresa).toEqual(wrapper.find('input').find({ name: 'nombreEmpresa' }).prop('value'));
    expect(datosPersonales.descripcionEmpresa).toEqual(
      wrapper.find('textarea').find({ name: 'descripcionEmpresa' }).prop('value')
    );
    expect(datosPersonales.sector.label).toEqual(wrapper.find({ name: 'sector' }).find('.select-big').text());
    expect(datosPersonales.giro.label).toEqual(wrapper.find({ name: 'giro' }).find('.select-big').text());
    expect(store.getState().solicitud.currentStep.paso).toBe(currentStep.paso + 1);
    expect(store.getState().solicitud.currentStep.valipStep).toBe(currentStep.valipStep + 1);
    expect(push).toHaveBeenCalledTimes(1);
    expect(push).toHaveBeenCalledWith(PASO_CUATRO_DATOS_PERSONA_ROUTE);
    wrapper.unmount();
  });

  test('Debe de guardar en el reducer el nuevo paso, nombreEmpresa, sector, giro, descripcionEmpresa y enviarlo al siguiente paso, como persona Moral', async () => {
    // arrange
    const store = storeTest();
    const {
      solicitud: { currentStep, datosPersonales },
    } = store.getState();
    datosPersonales.tipoPersona = MORAL;
    const wrapper = mount(
      <Provider store={store}>
        <PasoTresDatosPersonales sectores={sectores} validate />
        <ModalActualizar />
      </Provider>
    );

    await SectoresRepositorio.getGiroPorSector.mockResolvedValue({ data: giros });

    const form = wrapper.find('form');
    const razonSocial = wrapper.find('input').find({ name: 'razonSocial' });
    const tipoSociedad = wrapper.find({ name: 'tipoSociedad' }).find('.item');
    const nombreEmpresa = wrapper.find('input').find({ name: 'nombreEmpresa' });
    const rfcRepresentante = wrapper.find('input').find({ name: 'rfcRepresentante' });
    const sector = wrapper.find({ name: 'sector' }).find('.item');
    const descripcionEmpresa = wrapper.find('textarea').find({ name: 'descripcionEmpresa' });
    // act
    await act(async () => {
      razonSocial.simulate('change', {
        target: { value: 'paletitas', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      tipoSociedad.at(1).simulate('click');
    });

    await act(async () => {
      nombreEmpresa.simulate('change', {
        target: { value: 'Paletas don chucho', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      rfcRepresentante.simulate('change', {
        target: { value: 'CUPU800825569', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      descripcionEmpresa.simulate('change', {
        target: { value: 'Vende paletas', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      sector.at(1).simulate('click');
    });

    wrapper.update();
    const giro = wrapper.find({ name: 'giro' }).find('.item');

    await act(async () => {
      giro.at(1).simulate('click');
    });

    await act(async () => {
      form.simulate('submit');
    });

    const {
      solicitud: { datosPersonales: datosPersonalesAfter },
    } = store.getState();
    // // assert
    expect(datosPersonalesAfter.nombreEmpresa).toEqual(
      wrapper.find('input').find({ name: 'nombreEmpresa' }).prop('value')
    );
    expect(datosPersonalesAfter.descripcionEmpresa).toEqual(
      wrapper.find('textarea').find({ name: 'descripcionEmpresa' }).prop('value')
    );
    expect(datosPersonalesAfter.sector.label).toEqual(wrapper.find({ name: 'sector' }).find('.select-big').text());
    expect(datosPersonalesAfter.giro.label).toEqual(wrapper.find({ name: 'giro' }).find('.select-big').text());
    expect(store.getState().solicitud.currentStep.paso).toBe(currentStep.paso + 1);
    expect(store.getState().solicitud.currentStep.valipStep).toBe(currentStep.valipStep + 1);
    expect(push).toHaveBeenCalledTimes(1);
    expect(push).toHaveBeenCalledWith(PASO_CUATRO_DATOS_PERSONA_ROUTE);
    wrapper.unmount();
  });

  test('Debe de mantener el paso anterior en el reducer y actualizar los datos que se cambiaron y enviarlo al siguiente paso como persona fisica', async () => {
    // arrange
    const store = storeTest();
    const {
      solicitud: { currentStep, datosPersonales },
    } = store.getState();
    datosPersonales.tipoPersona = FISICA;

    const wrapper = mount(
      <Provider store={store}>
        <PasoTresDatosPersonales sectores={sectores} validate={false} />
        <ModalActualizar />
      </Provider>
    );

    await SectoresRepositorio.getGiroPorSector.mockResolvedValue({ data: giros });

    const form = wrapper.find('form');
    const nombreEmpresa = wrapper.find('input').find({ name: 'nombreEmpresa' });
    const sector = wrapper.find({ name: 'sector' }).find('.item');
    const descripcionEmpresa = wrapper.find('textarea').find({ name: 'descripcionEmpresa' });

    // act
    await act(async () => {
      nombreEmpresa.simulate('change', {
        target: { value: 'Paletas don chucho', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      descripcionEmpresa.simulate('change', {
        target: { value: 'Vende paletas', selectionStart: 0, setSelectionRange: jest.fn() },
      });
    });

    await act(async () => {
      sector.at(1).simulate('click');
    });

    wrapper.update();
    const giro = wrapper.find({ name: 'giro' }).find('.item');

    await act(async () => {
      giro.at(1).simulate('click');
    });

    await act(async () => {
      form.simulate('submit');
    });

    wrapper.update();

    await act(async () => {
      wrapper.find('.btn-medium-secondary').simulate('click');
    });

    const {
      solicitud: { datosPersonales: datosPersonalesAfter },
    } = store.getState();

    // assert
    expect(datosPersonalesAfter.nombreEmpresa).toEqual(
      wrapper.find('input').find({ name: 'nombreEmpresa' }).prop('value')
    );
    expect(datosPersonalesAfter.descripcionEmpresa).toEqual(
      wrapper.find('textarea').find({ name: 'descripcionEmpresa' }).prop('value')
    );
    expect(datosPersonalesAfter.sector.label).toEqual(wrapper.find({ name: 'sector' }).find('.select-big').text());
    expect(datosPersonalesAfter.giro.label).toEqual(wrapper.find({ name: 'giro' }).find('.select-big').text());
    expect(store.getState().solicitud.currentStep.paso).toBe(currentStep.paso);
    expect(store.getState().solicitud.currentStep.valipStep).toBe(currentStep.valipStep);
    expect(push).toHaveBeenCalledTimes(1);
    expect(push).toHaveBeenCalledWith(PASO_CUATRO_DATOS_PERSONA_ROUTE);
    wrapper.unmount();
  });
});
