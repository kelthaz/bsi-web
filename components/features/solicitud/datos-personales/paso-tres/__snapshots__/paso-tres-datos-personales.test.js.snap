// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente PasoTresDatosPersonales Debe de mostrarse correctamente el componente 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <PasoTresDatosPersonales
    sectores={
      Array [
        Object {
          "id": 1,
          "nombre": "SERVICIOS PROFESIONALES Y TÉCNICO",
        },
        Object {
          "id": 2,
          "nombre": "COMERCIO",
        },
        Object {
          "id": 3,
          "nombre": "SERVICIOS FINANCIEROS",
        },
        Object {
          "id": 4,
          "nombre": "CONSTRUCIÓN",
        },
        Object {
          "id": 5,
          "nombre": "RESTO INDUSTRIA",
        },
        Object {
          "id": 6,
          "nombre": "COMUNICACIONES Y TELECOMUNICACIONES",
        },
        Object {
          "id": 7,
          "nombre": "TRANSPORTE",
        },
        Object {
          "id": 8,
          "nombre": "HOTELES Y RESTAURANTES",
        },
        Object {
          "id": 9,
          "nombre": "INDUSTRIA AUTOMOTRIZ",
        },
        Object {
          "id": 10,
          "nombre": "INDUSTRIA QUIMICA Y FARMACÉUTICA",
        },
        Object {
          "id": 11,
          "nombre": "INDUSTRIA MATERIALES DE CONSTRUCCIÓN",
        },
        Object {
          "id": 12,
          "nombre": "INDUSTRIA TEXTIL Y DE CALZADO",
        },
        Object {
          "id": 13,
          "nombre": "SALUD",
        },
      ]
    }
    validate={true}
  >
    <div
      className="contedor-fixed-tab"
    >
      <div
        className="contedor-solicitud"
      >
        <div
          className="container p-0"
        >
          <form
            noValidate={true}
            onSubmit={[Function]}
          >
            <h2
              className="color-blue-storm"
            >
              ¡Anotado!
            </h2>
            <p
              className="color-dark-gray sub"
            >
              <span>
                ¿Cuál es el nombre comercial, sector y giro de tu negocio?
              </span>
            </p>
            <div
              className="row no-gutters"
            >
              <div
                className="col-lg-6 col-md-6 col-sm-12 col-xs-12"
              >
                <p
                  className="input color-gray"
                >
                  El nombre comercial es
                </p>
              </div>
              <div
                className="col-lg-6 col-md-6 col-sm-12 col-xs-12 pr-lg-2 pr-md-2"
              >
                <TextField
                  capitalize={false}
                  disabled={false}
                  format="uppercase"
                  formulario={
                    Object {
                      "dirty": false,
                      "errors": Object {},
                      "getFieldHelpers": [Function],
                      "getFieldMeta": [Function],
                      "getFieldProps": [Function],
                      "handleBlur": [Function],
                      "handleChange": [Function],
                      "handleReset": [Function],
                      "handleSubmit": [Function],
                      "initialErrors": Object {},
                      "initialStatus": undefined,
                      "initialTouched": Object {},
                      "initialValues": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                      "isSubmitting": false,
                      "isValid": true,
                      "isValidating": false,
                      "registerField": [Function],
                      "resetForm": [Function],
                      "setErrors": [Function],
                      "setFieldError": [Function],
                      "setFieldTouched": [Function],
                      "setFieldValue": [Function],
                      "setFormikState": [Function],
                      "setStatus": [Function],
                      "setSubmitting": [Function],
                      "setTouched": [Function],
                      "setValues": [Function],
                      "status": undefined,
                      "submitCount": 0,
                      "submitForm": [Function],
                      "touched": Object {},
                      "unregisterField": [Function],
                      "validateField": [Function],
                      "validateForm": [Function],
                      "validateOnBlur": true,
                      "validateOnChange": true,
                      "validateOnMount": false,
                      "values": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                    }
                  }
                  inverted={false}
                  label="Nombre del negocio"
                  maxlength={120}
                  name="nombreEmpresa"
                  optional={false}
                  paste={true}
                  readonly={false}
                  size="big"
                  type="text"
                >
                  <div
                    className="group"
                  >
                    <input
                      autoComplete="off"
                      className="input-big  "
                      disabled={false}
                      id="nombreEmpresa"
                      maxLength={120}
                      name="nombreEmpresa"
                      onBeforeInput={[Function]}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onKeyDown={[Function]}
                      onPaste={[Function]}
                      placeholder="Nombre del negocio"
                      readOnly={false}
                      tabIndex="0"
                      type="text"
                      value=""
                    />
                    <span>
                       
                    </span>
                  </div>
                </TextField>
              </div>
            </div>
            <div
              className="row no-gutters"
            >
              <div
                className="col-lg-4 col-md-4 col-sm-12 col-xs-12"
              >
                <p
                  className="input color-gray"
                >
                  el sector es de
                </p>
              </div>
              <div
                className="col-lg-8 col-md-8 col-sm-12 col-xs-12 pr-lg-2 pr-md-2"
              >
                <Select
                  blue={false}
                  disabled={false}
                  formulario={
                    Object {
                      "dirty": false,
                      "errors": Object {},
                      "getFieldHelpers": [Function],
                      "getFieldMeta": [Function],
                      "getFieldProps": [Function],
                      "handleBlur": [Function],
                      "handleChange": [Function],
                      "handleReset": [Function],
                      "handleSubmit": [Function],
                      "initialErrors": Object {},
                      "initialStatus": undefined,
                      "initialTouched": Object {},
                      "initialValues": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                      "isSubmitting": false,
                      "isValid": true,
                      "isValidating": false,
                      "registerField": [Function],
                      "resetForm": [Function],
                      "setErrors": [Function],
                      "setFieldError": [Function],
                      "setFieldTouched": [Function],
                      "setFieldValue": [Function],
                      "setFormikState": [Function],
                      "setStatus": [Function],
                      "setSubmitting": [Function],
                      "setTouched": [Function],
                      "setValues": [Function],
                      "status": undefined,
                      "submitCount": 0,
                      "submitForm": [Function],
                      "touched": Object {},
                      "unregisterField": [Function],
                      "validateField": [Function],
                      "validateForm": [Function],
                      "validateOnBlur": true,
                      "validateOnChange": true,
                      "validateOnMount": false,
                      "values": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                    }
                  }
                  inverted={false}
                  items={
                    Array [
                      Object {
                        "label": "SERVICIOS PROFESIONALES Y TÉCNICO",
                        "value": 1,
                      },
                      Object {
                        "label": "COMERCIO",
                        "value": 2,
                      },
                      Object {
                        "label": "SERVICIOS FINANCIEROS",
                        "value": 3,
                      },
                      Object {
                        "label": "CONSTRUCIÓN",
                        "value": 4,
                      },
                      Object {
                        "label": "RESTO INDUSTRIA",
                        "value": 5,
                      },
                      Object {
                        "label": "COMUNICACIONES Y TELECOMUNICACIONES",
                        "value": 6,
                      },
                      Object {
                        "label": "TRANSPORTE",
                        "value": 7,
                      },
                      Object {
                        "label": "HOTELES Y RESTAURANTES",
                        "value": 8,
                      },
                      Object {
                        "label": "INDUSTRIA AUTOMOTRIZ",
                        "value": 9,
                      },
                      Object {
                        "label": "INDUSTRIA QUIMICA Y FARMACÉUTICA",
                        "value": 10,
                      },
                      Object {
                        "label": "INDUSTRIA MATERIALES DE CONSTRUCCIÓN",
                        "value": 11,
                      },
                      Object {
                        "label": "INDUSTRIA TEXTIL Y DE CALZADO",
                        "value": 12,
                      },
                      Object {
                        "label": "SALUD",
                        "value": 13,
                      },
                    ]
                  }
                  label="Sector"
                  name="sector"
                  optional={false}
                  showAlwaysErrors={true}
                  size="big"
                >
                  <div
                    className="custom-select"
                  >
                    <button
                      className="svg-button-input-small arrow "
                      disabled={false}
                      onClick={[Function]}
                      tabIndex="-1"
                      type="button"
                    >
                      <SvgChevron>
                        <svg
                          fill="none"
                          height={22}
                          viewBox="0 0 12 22"
                          width={12}
                        >
                          <path
                            d="M1.667 1.667L11 11l-9.333 9.333"
                            stroke="#225AA7"
                            strokeLinecap="round"
                            strokeLinejoin="round"
                            strokeWidth={2}
                          />
                        </svg>
                      </SvgChevron>
                    </button>
                    <button
                      className="placeholder-color select-big indicador "
                      disabled={false}
                      onClick={[Function]}
                      tabIndex="0"
                      type="button"
                    >
                      Sector
                    </button>
                    <ul
                      className="select-items select-hide"
                      role="menu"
                    >
                      <li
                        key="1"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          SERVICIOS PROFESIONALES Y TÉCNICO
                        </button>
                      </li>
                      <li
                        key="2"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          COMERCIO
                        </button>
                      </li>
                      <li
                        key="3"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          SERVICIOS FINANCIEROS
                        </button>
                      </li>
                      <li
                        key="4"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          CONSTRUCIÓN
                        </button>
                      </li>
                      <li
                        key="5"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          RESTO INDUSTRIA
                        </button>
                      </li>
                      <li
                        key="6"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          COMUNICACIONES Y TELECOMUNICACIONES
                        </button>
                      </li>
                      <li
                        key="7"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          TRANSPORTE
                        </button>
                      </li>
                      <li
                        key="8"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          HOTELES Y RESTAURANTES
                        </button>
                      </li>
                      <li
                        key="9"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          INDUSTRIA AUTOMOTRIZ
                        </button>
                      </li>
                      <li
                        key="10"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          INDUSTRIA QUIMICA Y FARMACÉUTICA
                        </button>
                      </li>
                      <li
                        key="11"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          INDUSTRIA MATERIALES DE CONSTRUCCIÓN
                        </button>
                      </li>
                      <li
                        key="12"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          INDUSTRIA TEXTIL Y DE CALZADO
                        </button>
                      </li>
                      <li
                        key="13"
                        role="none"
                      >
                        <button
                          className="item "
                          onClick={[Function]}
                          role="menuitem"
                          tabIndex="-1"
                          type="button"
                        >
                          SALUD
                        </button>
                      </li>
                    </ul>
                    <span
                      className="help-text"
                    >
                      <span>
                         
                      </span>
                    </span>
                  </div>
                </Select>
              </div>
            </div>
            <div
              className="row no-gutters"
            >
              <div
                className="col-lg-4 col-md-4 col-sm-12 col-xs-12"
              >
                <p
                  className="input color-gray"
                >
                  y el giro es de
                </p>
              </div>
              <div
                className="col-lg-8 col-md-8 col-sm-12 col-xs-12 pr-lg-2 pr-md-2"
              >
                <Select
                  blue={false}
                  disabled={true}
                  formulario={
                    Object {
                      "dirty": false,
                      "errors": Object {},
                      "getFieldHelpers": [Function],
                      "getFieldMeta": [Function],
                      "getFieldProps": [Function],
                      "handleBlur": [Function],
                      "handleChange": [Function],
                      "handleReset": [Function],
                      "handleSubmit": [Function],
                      "initialErrors": Object {},
                      "initialStatus": undefined,
                      "initialTouched": Object {},
                      "initialValues": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                      "isSubmitting": false,
                      "isValid": true,
                      "isValidating": false,
                      "registerField": [Function],
                      "resetForm": [Function],
                      "setErrors": [Function],
                      "setFieldError": [Function],
                      "setFieldTouched": [Function],
                      "setFieldValue": [Function],
                      "setFormikState": [Function],
                      "setStatus": [Function],
                      "setSubmitting": [Function],
                      "setTouched": [Function],
                      "setValues": [Function],
                      "status": undefined,
                      "submitCount": 0,
                      "submitForm": [Function],
                      "touched": Object {},
                      "unregisterField": [Function],
                      "validateField": [Function],
                      "validateForm": [Function],
                      "validateOnBlur": true,
                      "validateOnChange": true,
                      "validateOnMount": false,
                      "values": Object {
                        "descripcionEmpresa": "",
                        "giro": null,
                        "nombreEmpresa": "",
                        "sector": null,
                      },
                    }
                  }
                  inverted={false}
                  items={Array []}
                  label="Giro"
                  name="giro"
                  optional={false}
                  showAlwaysErrors={true}
                  size="big"
                >
                  <div
                    className="custom-select"
                  >
                    <button
                      className="svg-button-input-small arrow "
                      disabled={true}
                      onClick={[Function]}
                      tabIndex="-1"
                      type="button"
                    >
                      <SvgChevron>
                        <svg
                          fill="none"
                          height={22}
                          viewBox="0 0 12 22"
                          width={12}
                        >
                          <path
                            d="M1.667 1.667L11 11l-9.333 9.333"
                            stroke="#225AA7"
                            strokeLinecap="round"
                            strokeLinejoin="round"
                            strokeWidth={2}
                          />
                        </svg>
                      </SvgChevron>
                    </button>
                    <button
                      className="placeholder-color select-big indicador "
                      disabled={true}
                      onClick={[Function]}
                      tabIndex="0"
                      type="button"
                    >
                      Giro
                    </button>
                    <ul
                      className="select-items select-hide"
                      role="menu"
                    />
                    <span
                      className="help-text"
                    >
                      <span>
                         
                      </span>
                    </span>
                  </div>
                </Select>
              </div>
            </div>
            <div
              className="row no-gutters"
            >
              <TextArea
                error=""
                format="textArea"
                initialTouched={false}
                initialValue=""
                label="Platícanos un poco a qué se dedica tu negocio..."
                maxlength={180}
                name="descripcionEmpresa"
                optional={false}
                setError={[Function]}
                setTouched={[Function]}
                setValue={[Function]}
                touched={false}
                value=""
              >
                <div
                  className="text-area-container"
                >
                  <div
                    className="border-text-area false"
                  >
                    <textarea
                      autoComplete="off"
                      className="text-area"
                      id="descripcionEmpresa"
                      maxLength={180}
                      name="descripcionEmpresa"
                      onBeforeInput={[Function]}
                      onBlur={[Function]}
                      onChange={[Function]}
                      placeholder="Platícanos un poco a qué se dedica tu negocio..."
                      tabIndex="0"
                      value=""
                    />
                    <div
                      className="resize-text-area"
                    />
                  </div>
                  <span
                    className="help-text"
                  >
                     
                  </span>
                </div>
              </TextArea>
            </div>
            <div
              className="flex-column-center-config"
            >
              <button
                aria-label="Avanzar"
                className="cicle-button-blue my-3"
                disabled={true}
                type="submit"
              />
            </div>
          </form>
        </div>
      </div>
    </div>
  </PasoTresDatosPersonales>
  <ModalActualizar>
    <Modal
      closeable={true}
      onClose={[Function]}
      openModal={false}
      setOpenModal={[Function]}
    />
  </ModalActualizar>
</Provider>
`;
